batch/plugins/org.jboss.tools.batch.core/templates/job.xml

```diff
@@ -0,0 +1,2 @@
+<job  id="myJob" xmlns="http://xmlns.jcp.org/xml/ns/javaee" version="1.0">
```

Comment:
1. Let's add `<?xml version="1.0" encoding="UTF-8"?>` too
2. Remove one extra space between `<job` and `id="myJob"`